python/python/tests/test_auth.py

```diff
@@ -66,6 +66,7 @@ async def test_update_connection_password(
         assert value == b"test_value"
         await glide_client.update_connection_password(None)
         await kill_connections(management_client)
+        await asyncio.sleep(1)
```

Comment:
why?

Reply:
We want to make sure the connections were already established.